content/vendor/helm/helm-processing.md

```diff
@@ -19,5 +19,79 @@ This level of change management provides the necessary transparency to provide t
 KOTS will check the apiVersion supplied in the Chart.yaml file of the Helm Chart to check if Helm V3 is necessary. By default (if Chart.yaml is not supplied or apiVersion is not present), KOTS will use Helm V2 to process all Helm Charts to create deployable YAML.
 Optionally, in the KOTS [`HelmChart`](https://kots.io/reference/v1beta1/helmchart/) resource an API version may be specified. When the `helmVersion` property is set to `"v3"`, KOTS will use Helm V3 to process the Helm Chart.
 
-Helm V3 integration is currently a beta feature. 
-In the future, KOTS may be updated to use Helm V3 by default.
+In the future as Helm V2 is removed from Helm support, KOTS may be updated to use Helm V3 by default.
```

Comment:
Helm v2 has already been removed from helm support: https://helm.sh/blog/helm-v2-deprecation-timeline/
I think we can just leave the line as it was.